tutorials/spa-create-automation/spa-create-automation.md

```diff
@@ -106,59 +106,80 @@ Business project usually need to use parameters and variable at runtime. These v
 
 You will now design the automation in the Automation Editor by dragging-and-dropping activities into the workflow of the automation. Later you will configure the inputs and outputs of each activity. You will need activities to interact with Microsoft Excel application. These activities will open Excel application, open the workbook that contains the sales orders details and map them into a data type that will be created during the design. Last, after extracting and mapping the data, the Excel application will be closed.
 
-1. Double click on the automation **Get Order Details** which navigates to the Design Studio to build your automation.
+1. Select 3 dots next to **Get Order Details**, choose **Open Editor**, which navigates to the Design Studio to build the automation.
```

Comment:
"Select 3 dots" may be a bit misleading. Also, it is better to spell out numbers up to 10. I would possibly say something like: "Choose the  icon with three dots next to..."

Reply:
in every tutorial we have term 3 dots. I can change to three